eclipse-scout-core/src/dataobject/dataObjects.ts

```diff
@@ -7,10 +7,54 @@
  *
  * SPDX-License-Identifier: EPL-2.0
  */
-
-import {arrays, DoEntity, objects, scout} from '..';
+import {
+  ArrayDoNodeSerializer, arrays, DateDoNodeSerializer, DoDeserializer, DoEntity, DoNodeSerializer, DoSerializer, doValueMetaData, IdDoNodeSerializer, MapDoNodeSerializer, objects, ObjectType, scout, SetDoNodeSerializer
+} from '../index';
 
 export const dataObjects = {
+
+  serializers: [
+    new DateDoNodeSerializer(),
```

Comment:
how do other layers/projects contribute new serializers? do they simply inject them into this array at the correct position?

Reply:
Yes